phpcs-ruleset.xml

```diff
@@ -2,4 +2,10 @@
 <ruleset name="Behat for WordPress coding standards">
   <rule ref="PSR1" />
   <rule ref="PSR2" />
+  <rule ref="PSR2.Methods.FunctionCallSignature.SpaceAfterCloseBracket">
```

Comment:
@paulgibbs PHPCS was missing the space violations in the first iteration of this PR. I've increased the severity so that they are caught in future. PR also fixes some of the errors that were raised as result.